### DISCUSSION TITLE
How to calculate the GND of BCT structure

### DISCUSSION THREAD

**Discussion by gskgs** [mtex-toolbox/mtex#2279] [Ask Anything]
Hi,
I'm trying to calculate GND for Sn which has a bct crystal structure, but the problem is that the dislocation systems are very well defined for BCC and FCC but not for bct systems. How do I define the slip system of BCT structure? 
Any help would be appreciated!
Thanks.

**Comment by kilir** [comment]
Hi, 
have a look here:

https://mtex-toolbox.github.io/slipSystem.slipSystem.html
You have to define the slip systems you want to use yourself.

Cheers,

Rüdiger
